staking/apr/compute.go

```diff
@@ -87,27 +94,31 @@ func ComputeForValidator(
 	)
 
 	if err != nil {
-		return &zero, err
+		return &zero, errors.Wrapf(
```

Comment:
don't need to accomdate the bad API from this package anymore, return nil,  the_wrapped_error. 

That's why we handled this, at callsite, in blockchain.go

Reply:
fall through will try to dereference if we return nil.

Reply:
then deal with it there. I want one place of ad-hoc code and that should be at call site, since we are doing it there on error. In this packages, like under staking/*, should assume the functions are pure in intent of computation asked, not business logic what should APR be when don't enough epoch

Reply:
there is still one case left, leave it as is? 
```
	if estimatedRewardPerYear.Cmp(common.Big0) == 0 {
		return &zero, nil
	}
```